### PR TITLE
openshot: use wrapQtAppsHook

### PR DESCRIPTION
**Pull request by emmanuelrosa**
###### Motivation for this change

This change fixes openshot-qt to account for the recent Qt changes; To use wrapQtAppsHook so Qt plugins can load.

I successfully created a project and exported an mp4 video.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Comment by AndersonTorres** [comment]
LGTM!

File: pkgs/applications/video/openshot-qt/default.nix [hunk 1]
```diff
@@ -13,18 +14,26 @@ python3Packages.buildPythonApplication rec {
     sha256 = "0mg63v36h7l8kv2sgf6x8c1n3ygddkqqwlciz7ccxpbm4x1idqba";
   };
 
-  nativeBuildInputs = [ doxygen wrapGAppsHook ];
+  nativeBuildInputs = [ doxygen wrapGAppsHook wrapQtAppsHook ];
```
Comment (ttuegel) [inline pkgs/applications/video/openshot-qt/default.nix]: `wrapQtAppsHook` is included by `mkDerivationWith`.

**Review by worldofpeace — APPROVED** [review]
Works for me, all feedback was addressed.

**Comment by worldofpeace** [comment]
Merged in f6b4c024cdb201ce0209af29a4c90b55384b2f72

**Pull request closed without merging**